packages/block-library/src/embed/index.js

```diff
@@ -138,21 +149,19 @@ export function getEmbedEdit( title, icon ) {
 		}
 
 		/**
-		 * Sets the appropriate CSS class to enforce an aspect ratio when the embed is resized
+		 * Gets the appropriate CSS class names to enforce an aspect ratio when the embed is resized
 		 * if the HTML has an iframe with width and height set.
 		 *
 		 * @param {string} html The preview HTML that possibly contains an iframe with width and height set.
+		 * @param {boolean} add If the classes should be added, or removed.
+		 * @return {Object} Object with classnames set for use with `classnames`.
 		 */
-		maybeSetAspectRatioClassName( html ) {
+		getAspectRatioClassNames( html, add = true ) {
```

Comment:
I'm cool with this approach, but I think this is a bit of a terse variable name. Maybe `includeAspectRatioClassnames`?

Sorry for all my suggestions being so verbose!

Reply:
how about `allowResponsive`? That's the same as the attribute name :)

Reply:
Works for me! 👍